nielsen/src/nielsen/Types.ts

```diff
@@ -129,6 +129,12 @@ export type NielsenDCRContentMetadataCZ = NielsenDCRContentMetadata & {
      * [LIVE] Keep empty : "nol_c2":"p2,"
      */
     c2?: string;
+    /*
+     * The actual duration of the live broadcast
+     * [VOD] Keep empty
+     * [LIVE] Pass the duration of the live broadcast in seconds
+     */
+    c4?: string;
```

Comment:
Since we have c1, c2 and now c4, is there also a c3 we should add? 😛

Reply:
😄 I thought about this too, but it seems to not be the case: 
<img width="918" alt="image" src="https://github.com/user-attachments/assets/e73f4d2f-51d7-40cc-8d8c-10305d16445e">